src/components/CommonComponents/Dropdown/index.tsx

```diff
@@ -24,12 +24,18 @@ const Dropdown = <T extends HTMLElement>({
       fontWeight: (item.active ? 'bold' : 'normal') as 'bold' | 'normal',
     };
 
+    const handleKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
+      if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
Any benefit to using key vs keycode https://github.com/w3c/aria-practices/blob/8e70c40029eca5328f68be949105ca6961235693/examples/button/js/button.js#L30

Reply:
Keycode is now deprecated so key is the suggested alternative

Reply:
Thanks, I thought one was depreciated, but I couldn't recall which